cmd/seimei/main.go

```diff
@@ -26,6 +28,27 @@ func main() {
 	}
 	flag.Parse()
 
+	if *file != "" {
+		data, err := os.ReadFile(*file)
```

Comment:
if we pass a huge file, i think it will occurs out of memory,  because this code load all data from file at once 💀

Reply:
I think so it.